org.metaborg.spoofax.shell.console/src/main/java/org/metaborg/spoofax/shell/client/console/impl/ConsoleReplModule.java

```diff
@@ -26,11 +27,17 @@
  */
 public class ConsoleReplModule extends ReplModule {
 
+    @Override
+    protected void configureCommands() {
+        super.configureCommands();
+        commandBinder.addBinding("exit").to(ExitCommand.class).in(Singleton.class);
```

Comment:
The `ExitCommand` is now tied to the implementation of `IRepl`, therefore, the current `ExitCommand` has been moved to the console module.